scripts/release-helpers/release-gen.sh

```diff
@@ -86,7 +86,7 @@ metadata.annotations.alm-examples: |-
       "spec": {
         "namespace": "kube-system",
         "secretRefName": "storageos-api",
-        "secretRefNamespace": "default",
+        "secretRefNamespace": "openshift-operators",
```

Comment:
If the secret is created in `openshift-operators` NS, shouldn't we move the cluster CR to the same NS?
All the resources in the same NS would be more consistent.